db/dcrsqlite/apisource.go

```diff
@@ -1262,3 +1262,56 @@ func (db *wiredDB) CountUnconfirmedTransactions(address string, maxUnconfirmedPo
 	}
 	return
 }
+
+// GetMepool gets all transactions from the mempool for explorer
```

Comment:
Sometime in the future we need to go through and remove all "Get" prefixes where it's absolutely not necessary.  This is good for now, just a heads up.